javaCode/angle.java

```diff
@@ -6,6 +6,7 @@ public static void main(String[] args) {
 
         rightTriangle();
         leftTriangle();
+        triangle();
```

Comment:
next time used functions that will create a diamond like asterisk. still nice job.